packages/quantic/force-app/main/default/lwc/quanticFacetManager/quanticFacetManager.js-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>true</isExposed>
```

Comment:
I'm not sure this component makes sense to be exposed in the lightning builder as it won't be possible to drag/drop it then facets inside of it.

Reply:
It would be VERY cool if we could craft components that could accept others to be dragged into it. I remember talking about this when thinking of creating a Case Assist `case information` component which could have inputs dragged into it.
Sadly AFAIK it is not supported. There's an [Idea](https://ideas.salesforce.com/s/idea/a0B8W00000GdnT1UAJ/drag-and-drop-regions-on-custom-lightning-components) for it and maybe some fan-made workarounds but i haven't seen anything ootb or simple to do.